docs/hardware/adding.md

```diff
@@ -439,9 +439,7 @@ JACK_TX needs to be on a pin with external IRQ and PAD0 of some SERCOM.
 
 JACK_SND needs to be on PA02 (DAC output).
 
-
-
-## Bootloaders
+### Bootloaders
```

Comment:
should this be something like 'existing bootloaders'? the section structure of Bootloader -> Bootloaders doesn't make sense to me